test/test_pipeline/components/data_preprocessing/test_one_hot_encoding.py

```diff
@@ -40,7 +42,7 @@ def test_default_configuration(self):
     def test_default_configuration_no_encoding(self):
         transformations = []
         for i in range(2):
-            transformation, original = _test_preprocessing(OneHotEncoder)
+            transformation, original = _test_preprocessing(NoEncoding)
```

Comment:
@mfeurer , I did change here, see if you agree.
Also, I quite don't understand how this test is passing in the master branch.